pkgs/tools/misc/fzf/default.nix

```diff
@@ -27,6 +27,11 @@ buildGoPackage rec {
     cp $src/bin/fzf-tmux $bin/bin
     mkdir -p $out/share/vim-plugins
     ln -s $out/share/go/src/github.com/junegunn/fzf $out/share/vim-plugins/${name}
```

Comment:
The issue here is a cycle between `$out` and `$bin` of the derivation for fzf (possibly an issue with the darwin stdenv). I don't think this will be fixed by those changes.
I'll test to confirm.